default_app/main.js

```diff
@@ -177,20 +177,33 @@ if (option.file && !option.webdriver) {
   if (!option.noHelp) {
     const welcomeMessage = `
 Electron ${process.versions.electron} - Build cross platform desktop apps with JavaScript, HTML, and CSS
-Usage: electron [options] [path]
+Usage: $electron [options] [path]
 
 A path to an Electron app may be specified. It must be one of the following:
   - index.js file.
+  - .html or .htm file.
+  - chrome://, http://, https://, or file:// URL.
   - Folder containing a package.json file.
   - Folder containing an index.js file.
-  - .html/.htm file.
-  - http://, https://, or file:// URL.
+
+When using a path that contains a package.json file, Electron will set:
+  - Set the appVersion
+  - Set the appName
+  - Set the userData directory path
+  - Set the userCache directory path
+  - Set the app path
+
+*IMPORTANT* FILE MODE: If you pass just a path that is in a directory without a package.json the app starts but doesn't set the above settings. This means many apis won't work properly.
```

Comment:
> This means many apis won't work properly.

This isn't entirely accurate, all APIs do work properly.

Reply:
I see, misinformed, I can remove this part no prob, any more ideas to add?